src/renderer/ui/components/generic/PlatformSpecific.js

```diff
@@ -1,14 +1,20 @@
 import { Component, PropTypes } from 'react';
+import os from 'os';
 
 export default class PlatformSpecific extends Component {
   static propTypes = {
     children: PropTypes.object,
     platform: PropTypes.string.isRequired,
+    version: PropTypes.string,
   };
 
   render() {
     if (process.platform === this.props.platform) {
-      return this.props.children;
+      if (!this.props.version) return this.props.children;
```

Comment:
@MarshallOfSound Yeah. I implemented it first using `semver` but it seemed a bit overkill to me.